### PR TITLE
TCOTA-447

### PR DESCRIPTION
**Pull request by isururanawaka**

### CODE REVIEW

File: src/main/javascript/src/views/outcome/OutcomeScoring.vue [hunk 1]
```diff
@@ -96,14 +96,16 @@ import {mapActions, mapGetters} from 'vuex'
       participantScoreList() {
         let arr = []
         const scoresAssociatedwithOutcome = this.outcomeScores.filter((score) => score.outcomeId === this.outcome_id)
-        this.participantFilteredList.map(p=>{
+        let sortedparticipantFilteredList = this.participantFilteredList;
```
Comment (machristie) [inline src/main/javascript/src/views/outcome/OutcomeScoring.vue:99]: this.participantFilteredList is already sorted, so I don't think it needs to be sorted again.

**Review by machristie — CHANGES_REQUESTED** [review]
Other than one small requested change, this looks good to me @isururanawaka .

File: src/main/javascript/src/views/outcome/OutcomeScoring.vue (hunk 1 above)
Comment (isururanawaka) [reply src/main/javascript/src/views/outcome/OutcomeScoring.vue]: yeah agreed. remove double sorting

**Review by machristie — APPROVED** [review]
LGTM 👍

**Pull request merged**